style_editor.html

```diff
@@ -982,7 +990,7 @@
 
   var str = new URL(window.location.href).searchParams.get("S");
   if (!str) {
-    str = "InOutHelper<SimpleClash<Lockup<Blast<Blue,White>,AudioFlicker<Blue,White>>,White>, 300, 800>";
+    str = "Layers<Red,ResponsiveLockupL<White,TrInstant,TrInstant,Int<26000>>,ResponsiveLightningBlockL<White>,ResponsiveMeltL<Mix<TwistAngle<>,Red,Yellow>>,ResponsiveDragL<White>,ResponsiveClashL<White,TrInstant,TrFade<200>,Int<26000>>,ResponsiveBlastL<White>,ResponsiveBlastWaveL<White>,ResponsiveBlastFadeL<White>,ResponsiveStabL<White>,InOutTrL<TrWipe<300>,TrWipeIn<500>>>";
```

Comment:
Unless this get automatically formatted into one line per layer at some point later, I think we should have the default be formatted properly.

Reply:
calling DoLayerize() in initGL seems to work.